schemas/com.unbounce/event_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for Unbounce event context (https://documentation.unbounce.com/hc/en-us/articles/203510044-Using-a-Webhook#content6)",
+  "self": {
+    "vendor": "com.unbounce",
+    "name": "event_context",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "page_id": {
+      "description": "The identifier Unbounce uses to uniquely identify your page. Eventually you’ll be able to use this with our API to manipulate and get information about your page. This is a 36-character UUID, for example, 'a2838d98-4cf4-11df-a3fd-00163e372d58'.",
+      "type": "string",
+      "maxLength": 36
+    },
+    "page_name": {
+      "description": "The name you gave your page, for example 'My Guaranteed to Convert Landing Page'.",
+      "type": "string"
+    },
+    "variant": {
+      "description": "This identifies the page variant that the visitor saw when they visited your page, and will be a lower-case letter. The first variant is 'a', the next, 'b', and so on. If you have more than 26 variants the sequence will continue with 'aa', 'ab', etc.",
+      "type": "string",
+      "maxLength": 8
+    },
+    "page_url": {
+      "description": "The URL of the page that contains your form.",
+      "type": "string",
+      "format": "uri",
+      "maxLength": 8192
+    },
+    "data.json": {
+      "type": "object",
+      "additionalProperties": true
+    },
+    "data.xml": {
+      "type": "string",
+      "description": "The submitted form data in XML format.",
+      "maxLength": 65535
+    }
```

Comment:
As far as I remember it just duplicates `data.json`?

Reply:
Correct.